### PR TITLE
Remove files

### PR DESCRIPTION
**Pull request by lina-ramirez-jalafund**
[AB#14](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/14)

# Overview

Delete folders
Change the class
Implement directives to Startup
Remove the DBConfig.json
Change the variables
Environment variables
Remove .editorconfig

### CODE REVIEW

File: src/api/Jalasoft.TeamUp.Resumes.API/Startup.cs [hunk 1]
```diff
@@ -19,7 +19,13 @@ public override void Configure(IFunctionsHostBuilder builder)
             builder.Services.AddScoped<IHealthService, HealthService>();
             builder.Services.AddScoped<IHealthRepository, HealthRepository>();
             builder.Services.AddScoped<IResumesService, ResumesService>();
-            builder.Services.AddScoped<IRepository<Resume>, ResumesRepository>();
+            builder.Services.AddScoped<IRepository<Resume>, ResumesInMemoryRepository>();
```
Comment (rodrigo-baldiviezo-jalafund) [inline src/api/Jalasoft.TeamUp.Resumes.API/Startup.cs]: this line is repeated in the line 27

Comment (lina-ramirez-jalafund) [reply src/api/Jalasoft.TeamUp.Resumes.API/Startup.cs]: Deleted line.

**Review by niver-copa-jalafund — DISMISSED** [review]

**Review by niver-copa-jalafund — DISMISSED** [review]

**Review by rodrigo-baldiviezo-jalafund — DISMISSED** [review]
LGTM

File: src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumeSQLRepository.cs [hunk 2]
```diff
@@ -10,14 +10,14 @@
...
     {
         private string connectionString;
 
         public ResumeSQLRepository()
         {
             IConfigurationRoot config = new ConfigurationBuilder()
-            .AddJsonFile("DBConfig.json", optional: false, reloadOnChange: true)
+            .AddJsonFile("local.settings.json", optional: false, reloadOnChange: true)
             .Build();
             this.connectionString = config["appSettings:ConnectionString"];
```
Comment (cristian-chavez-jalafund) [inline src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumeSQLRepository.cs]: I'm thinking if this is necessary? since the configuration variables are treated as environment variables we can do the same as we did in the project's repository to connect mongo, what do you think?
@alejandro-castellon-jalafund @rodrigo-baldiviezo-jalafund @niver-copa-jalafund

File: src/dal/Jalasoft.TeamUp.Resumes.DAL/Jalasoft.TeamUp.Resumes.DAL.csproj [hunk 3]
```diff
@@ -21,10 +21,4 @@
 	  <ProjectReference Include="..\Jalasoft.TeamUp.Resumes.DAL.Interfaces\Jalasoft.TeamUp.Resumes.DAL.Interfaces.csproj" />
```
Comment (cristian-chavez-jalafund) [inline src/dal/Jalasoft.TeamUp.Resumes.DAL/Jalasoft.TeamUp.Resumes.DAL.csproj:21]: I comment here since I can't do it above. In lines 10 and 11 please double-check this are necessary otherwise remove them

File: src/api/Jalasoft.TeamUp.Resumes.API/local.settings.json [hunk 4]
```diff
@@ -6,5 +6,9 @@
   },
   "Host": {
     "CORS": "*"
+  },
+
+  "appSettings": {
+    "connectionString": "Server = DESKTOP-L33PS1L; DataBase = TeamUp; User ID = AppConnection; Password=12345678; "
```
Comment (Raul-Gamarra-jalafund) [inline src/api/Jalasoft.TeamUp.Resumes.API/local.settings.json]: Instead of creating appSettings please insert the SQL connections string in the Values, just like we did in Projects

File: src/dal/Jalasoft.TeamUp.Resumes.DAL/Jalasoft.TeamUp.Resumes.DAL.csproj (hunk 3 above)
Comment (lina-ramirez-jalafund) [reply src/dal/Jalasoft.TeamUp.Resumes.DAL/Jalasoft.TeamUp.Resumes.DAL.csproj:21]: Remove them.

**Review by niver-copa-jalafund — DISMISSED** [review]

File: src/api/Jalasoft.TeamUp.Resumes.API/local.settings.json (hunk 4 above)
Comment (lina-ramirez-jalafund) [reply src/api/Jalasoft.TeamUp.Resumes.API/local.settings.json]: Insert it.

File: src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumeSQLRepository.cs (hunk 2 above)
Comment (lina-ramirez-jalafund) [reply src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumeSQLRepository.cs]: Changing environment variables.

Comment (alejandro-castellon) [reply src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumeSQLRepository.cs]: yes too much easier

Comment (rodrigo-baldiviezo-jalafund) [reply src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumeSQLRepository.cs]: It´s other way to do. I agree with both

**Review by alejandro-castellon — DISMISSED** [review]

File: .editorconfig [hunk 5]
```diff
@@ -0,0 +1,4 @@
+﻿[*.{cs,vb}]
```
Comment (Raul-Gamarra-jalafund) [inline .editorconfig]: why was this file added?

Comment (rodrigo-baldiviezo-jalafund) [reply .editorconfig]: same question

Comment (lina-ramirez-jalafund) [reply .editorconfig]: A mistake, sorry.

**Review by niver-copa-jalafund — DISMISSED** [review]

File: src/core/Jalasoft.TeamUp.Resumes.Core.Interfaces/Jalasoft.TeamUp.Resumes.Core.Interfaces.csproj [hunk 6]
```diff
@@ -5,6 +5,10 @@
 		<CodeAnalysisRuleSet>../../../stylecop.ruleset</CodeAnalysisRuleSet>
 	</PropertyGroup>
 
+	<ItemGroup>
+	  <None Include="..\..\..\.editorconfig" Link=".editorconfig" />
+	</ItemGroup>
+
```
Comment (rodrigo-baldiviezo-jalafund) [inline src/core/Jalasoft.TeamUp.Resumes.Core.Interfaces/Jalasoft.TeamUp.Resumes.Core.Interfaces.csproj]: this file exists yet?

File: resumes-api.sln [hunk 7]
```diff
@@ -35,6 +35,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Jalasoft.TeamUp.Resumes.API
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Jalasoft.TeamUp.Resumes.Core.Tests", "tests\core\Jalasoft.TeamUp.Resumes.Core.Tests\Jalasoft.TeamUp.Resumes.Core.Tests.csproj", "{B940D2ED-661C-4904-8782-5061F1600BD9}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{C585EB76-AD2C-40D1-869C-497E7EF36288}"
+	ProjectSection(SolutionItems) = preProject
+		.editorconfig = .editorconfig
+	EndProjectSection
+EndProject
```
Comment (rodrigo-baldiviezo-jalafund) [inline resumes-api.sln]: this file exists yet?

File: src/core/Jalasoft.TeamUp.Resumes.Core.Interfaces/Jalasoft.TeamUp.Resumes.Core.Interfaces.csproj (hunk 6 above)
Comment (lina-ramirez-jalafund) [reply src/core/Jalasoft.TeamUp.Resumes.Core.Interfaces/Jalasoft.TeamUp.Resumes.Core.Interfaces.csproj]: Remove.

File: resumes-api.sln (hunk 7 above)
Comment (lina-ramirez-jalafund) [reply resumes-api.sln]: Remove.

**Review by rodrigo-baldiviezo-jalafund — APPROVED** [review]
LGTM

**Review by niver-copa-jalafund — APPROVED** [review]

**Review by Raul-Gamarra-jalafund — APPROVED** [review]
